tests/integration/testlib/assertlib.py

```diff
@@ -23,16 +23,38 @@
 from .statelib import INTERFACES
 
 
+def _is_ipv6_link_local(ip, prefix):
+    """
+    Just do simple string starts with check and prefix check, as this is purely
+    for test only.
+    """
+    return ip.startswith("fe80") and prefix >= 10
```

Comment:
How about checking if it starts with `fe8`, `fe9`, `fea` or `feb` with this prefix size, then it should catch all link-local addresses?

Reply:
Good point. Thank you.